src/main/config/work/staging/sys/batchContainer/placeholder.txt

```diff
@@ -0,0 +1,105 @@
+
+  3.
+
+  Fade far away, dissolve, and quite forget
+    What thou among the leaves hast never known,
+  The weariness, the fever, and the fret
+    Here, where men sit and hear each other groan;
+  Where palsy shakes a few, sad, last gray hairs,
+    Where youth grows pale, and spectre-thin, and dies;
+      Where but to think is to be full of sorrow
+          And leaden-eyed despairs,
+    Where Beauty cannot keep her lustrous eyes,
+      Or new Love pine at them beyond to-morrow.               30
+
+  4.
+
+  Away! away! for I will fly to thee,
+    Not charioted by Bacchus and his pards,
+  But on the viewless wings of Poesy,
+    Though the dull brain perplexes and retards:
+  Already with thee! tender is the night,
+    And haply the Queen-Moon is on her throne,
+      Cluster'd around by all her starry Fays;
+          But here there is no light,
+    Save what from heaven is with the breezes blown
+      Through verdurous glooms and winding mossy ways.         40
+
+  5.
+
+  I cannot see what flowers are at my feet,
+    Nor what soft incense hangs upon the boughs,
+  But, in embalmed darkness, guess each sweet
+    Wherewith the seasonable month endows
+  The grass, the thicket, and the fruit-tree wild;
+    White hawthorn, and the pastoral eglantine;
+      Fast fading violets cover'd up in leaves;
+          And mid-May's eldest child,
+    The coming musk-rose, full of dewy wine,
+      The murmurous haunt of flies on summer eves.             50
+
+  6.
+
+  Darkling I listen; and, for many a time
+    I have been half in love with easeful Death,
+  Call'd him soft names in many a mused rhyme,
+    To take into the air my quiet breath;
+  Now more than ever seems it rich to die,
+    To cease upon the midnight with no pain,
+      While thou art pouring forth thy soul abroad
+          In such an ecstasy!
+    Still wouldst thou sing, and I have ears in vain--
+      To thy high requiem become a sod.                        60
+
+  7.
+
+  Thou wast not born for death, immortal Bird!
+    No hungry generations tread thee down;
+  The voice I hear this passing night was heard
+    In ancient days by emperor and clown:
+  Perhaps the self-same song that found a path
+    Through the sad heart of Ruth, when, sick for home,
+      She stood in tears amid the alien corn;
+          The same that oft-times hath
+    Charm'd magic casements, opening on the foam
+      Of perilous seas, in faery lands forlorn.                70
+
+  8.
+
+  Forlorn! the very word is like a bell
+    To toll me back from thee to my sole self!
+  Adieu! the fancy cannot cheat so well
+    As she is fam'd to do, deceiving elf.
+  Adieu! adieu! thy plaintive anthem fades
+    Past the near meadows, over the still stream,
+      Up the hill-side; and now 'tis buried deep
+          In the next valley-glades:
+    Was it a vision, or a waking dream?
+      Fled is that music:--Do I wake or sleep?                 80
+
+-- John Keats
```

Comment:
so, I'm just noticing this - it looks like these files were added to the new location, but not moved/deleted from the old location? am I misreading?